lms/models/grouping.py

```diff
@@ -160,6 +160,20 @@ class D2LGroup(Grouping):
 class Course(Grouping):
     __mapper_args__ = {"polymorphic_identity": Grouping.Type.COURSE}
 
+    def get_mapped_file_id(self, file_id):
```

Comment:
Moved from the service. `extra` is part of the model and I'd prefer to just take a look here to see what possibly could `extra` contain.